src/uu/ls/src/ls.rs

```diff
@@ -1992,3 +2086,47 @@ fn display_symlink_count(_metadata: &Metadata) -> String {
 fn display_symlink_count(metadata: &Metadata) -> String {
     metadata.nlink().to_string()
 }
+
+// This returns the SELinux security context as UTF8 `String`.
+// In the long term this should be changed to `OsStr`, see discussions at #2621/#2656
+fn get_security_context(config: &Config, p_buf: &PathBuf, must_dereference: bool) -> String {
```

Comment:
you have some warnings here :)

Reply:
Feels free to silent the warning if it increases the complexity

Reply:
done